source/clear-linux/reference/collaboration/community/irc.rst

```diff
@@ -0,0 +1,13 @@
+.. _irc:
+
+IRC channel
+###########
+
+The |CLOSIA| team uses an IRC channel for quick chat-like communication. You
+can find it as #clearlinux. You can access the channel through the freenode\*
+`webchat`_ or via IRC applications such as `Pidgin`_\*. Setting up your IRC
```

Comment:
I don't think we need to state that setting up your IRC client is out of scope. I suggest we delete the sentence (preferred), or add a link to where you can get info on getting started with irc.

Reply:
Removed the sentence as preferred.